packages/api/ecs/async-operation/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:8.10
+FROM node:10.16.3
```

Comment:
We'll need to release an updated version of the `async-operation` image to Docker Hub.

Reply:
Done, and updated 4095166